include/os/linux/spl/sys/isa_defs.h

```diff
@@ -223,10 +223,34 @@
 
 #define	_ALIGNMENT_REQUIRED	1
 
+/*
+ * loongarch arch specific defines
+ * only loongarch64 is supported yet
+ */
+
+#elif defined(__loongarch__)
+
+#if !defined(__loongarch64)
+#define	__loongarch64
+#endif
+
+#define	_ZFS_LITTLE_ENDIAN
+#define	_SUNOS_VTOC_16
+
+#ifndef _LP64
+#define	_LP64
+#endif
+
+/*
+ * Illumos doesn't define _ALIGNMENT_REQUIRED for loongarch, so default to 1
+ * out of paranoia.
+ */
```

Comment:
There isn't a LoongArch port of Illumos (yet), so better reword the comment. The actual reason why we really want to require alignment is that LoongArch implementations are explicitly not required to support unaligned accesses by the ISA spec.